packages/cli/src/commands/dkg/deploy.ts

```diff
@@ -0,0 +1,31 @@
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { displayWeb3Tx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+const DKG = require('./DKG.json')
+
+export default class DKGDeploy extends BaseCommand {
+  static description = 'Deploys the DKG'
```

Comment:
You could extend a little the description to avoid possible mistakes from users that don't know what is this